wscript

```diff
@@ -376,7 +376,12 @@ configuration in order to save typing.
     g.add_option('--assert-cc-version',
                  default=None,
                  help='fail configure if not using the specified gcc version')
-    
+
+    g.add_option('--num-aux-imus',
```

Comment:
we need a sim_vehicle.py option for this to make testing easier, or even better, just use the existing SIM_IMU_COUNT and make it runtime like it is now

Reply:
@tridge We do want to retain the check with INS_MAX_COUNT as 3 by default, moving the limit to SIM_IMU_COUNT will require raising INS_MAX_COUNT to 5.

Reply:
I have added the option to sim_vehicles.py